trulens_eval/trulens_eval/feedback/benchmark_frameworks/meta_evaluation_examples.ipynb

```diff
@@ -0,0 +1,305 @@
+    "        \"response\": \"Paris is the capital of France.\",\n",
+    "        \"expected_score\": 1.0,\n",
+    "    },\n",
+    "    {\n",
+    "        \"query\": \"what is the capital of Spain?\",\n",
+    "        \"response\": \"I love going to Spain.\",\n",
+    "        \"expected_score\": 0,\n",
+    "    },\n",
+    "]\n",
+    "# Create a Feedback object using the numeric_difference method of the ground_truth object\n",
+    "ground_truth = GroundTruthAgreement(golden_set)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from trulens_eval.feedback import Cortex\n",
+    "\n",
+    "provider = Cortex(model_engine=\"mistral-large\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from typing import Tuple\n",
+    "\n",
+    "\n",
+    "# output is feedback_score\n",
+    "def context_relevance_ff_to_score(input, output, benchmark_params) -> float:\n",
+    "    return provider.context_relevance(\n",
+    "        question=input,\n",
+    "        context=output,\n",
+    "        temperature=benchmark_params[\"temperature\"],\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "# output is (feedback_score, confidence_score)\n",
+    "def context_relevance_ff_to_score_with_confidence(\n",
+    "    input, output, benchmark_params\n",
+    ") -> Tuple[float, float]:\n",
+    "    return provider.context_relevance_verb_confidence(\n",
+    "        question=input,\n",
+    "        context=output,\n",
+    "        temperature=benchmark_params[\"temperature\"],\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Collect all prompt and expected responses from the golden set and pass to GroundTruthAggregator as ground truth labels"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "prompts = []\n",
+    "responses = []\n",
+    "for i in range(len(golden_set)):\n",
+    "    prompt = golden_set[i][\"query\"]\n",
+    "    response = golden_set[i][\"response\"]\n",
+    "\n",
+    "    prompts.append(prompt)\n",
+    "    responses.append(response)\n",
+    "\n",
+    "true_labels = [entry[\"expected_score\"] for entry in golden_set]\n",
+    "\n",
+    "mae_agg_func = GroundTruthAggregator(true_labels=true_labels).mae"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from trulens_eval.feedback.benchmark_frameworks.tru_benchmark_experiment import (\n",
+    "    BenchmarkParams,\n",
+    ")\n",
+    "\n",
+    "tru_benchmark_arctic = tru.BenchmarkExperiment(\n",
+    "    app_id=\"MAE\",\n",
+    "    ground_truth=golden_set,\n",
+    "    trace_to_score_fn=context_relevance_ff_to_score,\n",
+    "    agg_funcs=[mae_agg_func],\n",
```

Comment:
I'm confused by the SDK, like we know
```
mae_agg_func = GroundTruthAggregator(true_labels=true_labels).mae
```
but why do we have to say the true labels in `mae_agg_func` and in the arg `ground_truth` to `tru.BenchmarkExperiment`? It seems kinda redundent.

Reply:
removed redundant argument in this [commit](https://github.com/truera/trulens/pull/1282/commits/08ff34985a60610061aeebd2ec4d341812aa305d)